docs/sources/panels/visualizations/geomap.md

```diff
@@ -8,4 +8,123 @@ weight = 600
 
 # Geomap
 
-The Geomap panel visualization allows you to view ...
\ No newline at end of file
+{{< figure src="/static/img/docs/geomap-panel/geomap_example.png" max-width="1200px" caption="Geomap panel" >}}
+
+The Geomap panel visualization allows you to view and customize the world map using geospatial data in respect to geographical locations. Various overlay styles and map view settings can be configured to easily focus on the important location-based characteristics of the data.
+
+
+## Base layer
+
+Base layer loads in a blank world map from the tile server to the Grafana panel. There are several base layer options available for you to choose from, each with their specific configuration options to style the base map. Default base layer is CartoDB base map, but can be specified differently through the `.ini` configuration file.
```

Comment:
Suggestion: `Base layer loads in a blank world map from the tile server to the Grafana panel. There are several base layer options available for you to choose from, each with specific configuration options to style the base map. The default base layer is CartoDB base map. Custom default base layers can be defined using the `.ini` configuration file.